src/tag.js

```diff
@@ -57,5 +57,5 @@ export default function tag(fn: Command | Tagged, name?: string): Tagged {
   // Mark the function as tagged so we only do this once
   fn.__tagged = true
 
-  return cast
+  return fn
```

Comment:
Really happy about this. It looks like you can cast a Function type to an Object in flow to assign properties to it:

```
  let cast: Tagged = fn

  // assign props
  cast.open = symbol + '.open'
  // ...

  // Return the original function to make flow happy
  return fn
```